### DISCUSSION TITLE
Multiple "where" syntax of redbean API

### DISCUSSION THREAD

**Discussion by TehoChang** [Lidemy/forum#212] [後端相關問題]
工作上碰到的問題，卡了很久，所以來這邊提問，若不合適的我會刪除，謝謝
發送request給一個redbean API，request.body為以下
```
{
    "action": "ELEGANT:searchOrderByShipref",
    "compid": 117,
    "shipref": "",
    "record_type": ""
}
```
假設"record_type"的值可以為"a", "b" ,"c"，能分別撈回三組資料。若我今天想撈回a or b, b or c或是全部撈回，應該怎麼寫呢？
嘗試了以下方法都失敗：
1 刪除record_type這個field
2 寫  "record_type": "" （空字串）

如果在資料庫，使用
`WHERE id = 3 or id = 4`
Or 
`WHERE id in (3,4)`
可以完成需求

如何才能查到這個API的語法、使用方式呢？ google： redbean multiple where syntax, select multiple values ...找不到正確答案
懇請高手指教，謝謝！

**Comment by aszx87410** [comment]
redbean API 是你用的某個套件嗎還是什麼？這看起來有可能是後端要處理的，不是前端

**Reply by TehoChang** [reply]
是一套php的後端"框架"嗎，寫出來的api都用同一個endpoint，改變參數來撈回各種資料
https://redbeanphp.com/index.php

**Reply by TehoChang** [reply]
謝謝老師指導的方向，確實要從後端著手查詢，前端瞎測瞎試解決不了問題。
